frontend/src/layouts/Layout.jsx

```diff
@@ -19,65 +24,68 @@ const Layout = ({ crumbs }) => {
   return (
-    <RequireAuth redirectTo={appRoutes.LOGIN}>
+    <RequireAuth redirectTo={appRoutes.login.main}>
+      <Navbar />
       <Container
         maxWidth="lg"
         sx={({ palette: { background } }) => ({
           padding: '1rem',
+          minHeight: 'calc(100vh - 4.89rem)',
           background: background.paper,
           '@media (max-width: 920px)': {
             marginTop: '-2rem'
           }
         })}
         disableGutters={true}
       >
-        <Grid
-          item
-          xs={12}
-          sx={{
-            maxHeight: '20vh',
-            position: 'relative',
-            top: 0,
-            zIndex: 10 // Adjust the z-index if needed
-          }}
-        >
-          <Navbar />
+        <Grid item xs={12} mt={12}>
+          {crumbs && (
+            <Breadcrumbs
+              m={2}
+              separator={
+                <NavigateNextIcon fontSize="small" aria-label="breadcrumb" />
+              }
+            >
+              {breadcrumbs.map((crumb, index) =>
+                index + 1 !== breadcrumbs.length ? (
+                  <BCTypography
+                    key={crumb.path}
+                    component={RouterLink}
+                    to={crumb.path}
+                    disabled={true}
+                    variant="button"
+                    fontSize={size.lg}
+                  >
+                    {crumb.label}
+                  </BCTypography>
+                ) : (
+                  <BCTypography
+                    variant="button"
+                    key={crumb.path}
+                    color="text"
+                    fontSize={size.lg}
+                  >
+                    {crumb.label}
+                  </BCTypography>
+                )
+              )}
+            </Breadcrumbs>
+          )}
         </Grid>
-        <Grid item my={12} lg={12}>
-          <BCBox>
-            {crumbs && (
-              <Paper
-                p={2}
-                elevation={5}
-                sx={{ padding: '8px', height: '50px' }}
-              >
-                <Breadcrumbs separator={<NavigateNextIcon fontSize="small" />}>
-                  {breadcrumbs.map((crumb, index) =>
-                    index + 1 !== breadcrumbs.length ? (
-                      <BCTypography
-                        key={crumb.path}
-                        component={RouterLink}
-                        to={crumb.path}
-                        disabled={true}
-                      >
-                        {crumb.label}
-                      </BCTypography>
-                    ) : (
-                      <BCTypography key={crumb.path}>
-                        {crumb.label}
-                      </BCTypography>
-                    )
-                  )}
-                </Breadcrumbs>
-              </Paper>
-            )}
-            <BCBox py={4}>
-              <Outlet />
-              <Footer />
-            </BCBox>
-          </BCBox>
+        <Grid item lg={12}>
```

Comment:
the item prop is throwing a console warning. we can probably remove it

Reply:
yes, removed.